### DISCUSSION TITLE
Whether the older version will have a splash screen

### DISCUSSION THREAD

**Discussion by xueyuebingying** [airsdk/Adobe-Runtime-Support#822] [Q&A] [answered]
If I use 32 previous adobe air.
Whether the harman splash screen will appear.

**Comment by ajwfrost** [comment, marked as answer]
The splash screen functionality is built into the AIR 33.1 runtime. So, if you have got Adobe's AIR v32, you won't ever see any splash screen.

AIR v32 can continue to be used on desktop platforms, I believe there are some extra hoops to jump through in order to get it working properly on MacOS Catalina and beyond. But also note that this version won't be around forever, Adobe will take this down once it's no longer required which means for a runtime you'd need to direct people to the Harman version (all redistribution licenses for the shared AIR runtime have expired).

Also please note that the ability to remove the splash screen from AIR v33 is only something you get with a commercial SDK license. So if you have an old AIR application and can't update it, then you'll see a splash screen when running it within the AIR 33.1 shared runtime from Harman.

Hope that helps to clarify the approach!

thanks

**Reply by xueyuebingying** [reply]
Thank you for your reply

**Comment by teotigraphix** [comment]
@ajwfrost Are you aware of this? How is this going to impact your Splash Screen?

https://forum.starling-framework.org/d/23032-android-12-splash-screens-oh-my

**Reply by ajwfrost** [reply]
Means we have to rework things, again :-( but I believe it means that it will actually look a little better once done!
So perhaps more like on iOS, we can have a way for people to define their splash screen, but for "free tier" folk we'll need to enforce this to be our own one.

thanks
